src/include/distributed/metadata_utility.h

```diff
@@ -30,7 +30,7 @@
 
 /* total number of hash tokens (2^32) */
 #define HASH_TOKEN_COUNT INT64CONST(4294967296)
-#define SELECT_EXIST_QUERY "SELECT EXISTS (SELECT 1 FROM %s)"
+#define SELECT_EXIST_QUERY "SELECT EXISTS (SELECT TRUE FROM %s LIMIT 1)"
```

Comment:
maybe remove the `SELECT EXISTS (`. Otherwise, this query still does some extra steps.